extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -257,7 +268,15 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
     this.inputElement_.setAttribute('dir', 'auto');
     this.inputElement_.setAttribute('aria-autocomplete', 'both');
-    this.inputElement_.setAttribute('role', 'combobox');
+    this.inputElement_.setAttribute('role', 'textbox');
```

Comment:
Nit: Is this `inputElement_` constructed in this callback? If you can use `createElementWithAttributes` here you'd save a few bytes.

Reply:
No, it is provided by the publisher.